### PR TITLE
feat: updated libraries to use metamask in mobile

### LINKED ISSUE #430: When seller is not owner no error is shown
Probably related to https://github.com/decentraland/marketplace/issues/178 , some people is trying to buy this land, but Memamask don't show up, and no error is shown.

On the console the error is: `MetaMask - RPC Error: execution reverted: The seller is no longer the owner`

(-61,150)

https://market.decentraland.org/contracts/0xf87e31492faf9a91b02ee0deaad50d51d56d5d4d/tokens/115792089237316195423570985008687907832512760283463317768191732954575268741270

### PR DESCRIPTION
**Pull request by flobarreto**
closes #430

### CODE REVIEW

**Comment by flobarreto** [comment]
image

File: webapp/package.json [hunk 1]
```diff
@@ -101,4 +101,4 @@
     ]
   },
   "homepage": ""
-}
+}
```
Comment (juanmahidalgo) [inline webapp/package.json:104]: can we add this end of file back please? 🙏

**Review by juanmahidalgo — APPROVED** [review]
LGTM!

**Review by juanmahidalgo — APPROVED** [review]
LGTM!

**Pull request merged**